lib/base/application.cpp

```diff
@@ -180,7 +180,7 @@ void Application::SetResourceLimits(void)
 	rlimit rl;
 
 #	ifdef RLIMIT_NOFILE
-	rl.rlim_cur = 16 * 1024;
+	rl.rlim_cur = GetRLimitFiles();
```

Comment:
Ah ok. Is there a sensible limit for these three options if the user puts in garbage, or like 10 files?